app/models/miq_action.rb

```diff
@@ -829,6 +829,26 @@ def action_ems_refresh(action, rec, inputs)
     end
   end
 
+  def action_container_image_analyze(action, rec, inputs)
+    action_method = "action_container_image_analyze"
```

Comment:
Can you use `__method__` instead?  It should be the same thing but as a symbol:  http://ruby-doc.org/core-2.3.0/Kernel.html#method-i-__method__